dcm4che-imageio-test/src/test/java/org/dcm4che3/imageio/dcm/TestDicomImageReader.java

```diff
@@ -136,7 +137,15 @@ public void testReadUncompressedPostPixelData_fromImageInputStream() throws IOEx
             testReadPostPixelData(is);
         }
     }
-    
+
+    @Test
+    public void testReadUncompressedSingleFrame_fromImageInputStream() throws IOException {
```

Comment:
@wayfarer3130, are you sure that this test is actually covering the scenario that you fixed?
I reverted you fixes, ran the test, and it is still succeeding.

(I just noticed this by coincidence, because I am playing around with jacoco code coverage and noticed that this test is not even running through any of the lines that you have modified.)

Reply:
Same as @hczedik. This test is working before the changes.
@wayfarer3130  could you provide a way to get an OOM ?

Reply:
Fixed the test - I had been seeing a couple of exceptions in some other code I was working on for my project static-wado, and thought it was the uncompressed case causing the issues, but in fact it was a compressed single frame issue.  Strangely, it does NOT occur with JPEG-LS, not sure what is different, but does occur on JPLY, for which we have a test image already.  Updated that, confirmed the issue fails without the fixes and passes with the fixes.